src/moonstone-samples/lib/LightPanelsSample.js

```diff
@@ -38,7 +61,14 @@ module.exports = kind({
 				}
 			],
 			clientComponents: [
-				{kind: Scroller, style: 'height:800px', components: [
+				{kind: Control, classes: 'control-container', components: [
+					{kind: Control, components: [
+						{kind: IconButton, name: 'previous', icon: 'arrowlargedown', ontap: 'prevIconTapped'},
```

Comment:
The `previous` and `next` names conflict when you add additional panels. Doesn't appear to need to reference them so let's remove the names.